docs/raw/9-blockchain-interactions.md

```diff
@@ -0,0 +1,186 @@
+---
+permalink: /spec/9
+parent: Raw specs
+title: 9/BLOCKCHAIN-USAGE
+---
+
+# 3/BLOCKCHAIN-USAGE
+
+> Version: 0.1
+>
+> Status: Raw
+>
+> Authors: Andrea Maria Piana <andreap@status.im>
+
+# Status interactions with the blockchain
```

Comment:
I'd be explicit with this being Ethereum and a link to the specs we rely on here.